applications/nrf_desktop/README.rst

```diff
@@ -949,7 +949,7 @@ To use the nRF Desktop application with your custom board:
      * Each of the logical LEDs can have either one (monochromatic) or three color channels (RGB).
        Such color channel is a physical LED.
      * The project uses Pulse-Width Modulation (PWM) channels to control the brightness of each physical LED.
-       The PWM peripheral must be configured in DTS files, and the :file:`_def` file of the LEDs module must be configured to indicate which PWM channel is assigned to each LED color.
+       Configure the PWM peripheral in DTS files, and the :file:`_def` file of the LEDs module must be configured to indicate which PWM channel is assigned to each LED color.
```

Comment:
This is now in active mode in first part and passive mode in 2nd  :) 

```suggestion
       Configure the PWM peripheral in DTS files, and configure the :file:`_def` file of the LEDs module to indicate which PWM channel is assigned to each LED color.
```

Reply:
Good catch. I noticed this myself, but left it for you ;)